lxd/entity/type.go

```diff
@@ -3,10 +3,12 @@ package entity
 import (
 	"fmt"
 	"net/url"
+	"runtime"
```

Comment:
the commit message mentions shared/entity still

Reply:
Ah I missed that. Fixed now.